lib/view/profiel/ProfielForm.php

```diff
@@ -160,7 +160,7 @@ public function __construct(Profiel $profiel) {
 		$fields[] = new RequiredTextField('woonplaats', $profiel->woonplaats, 'Woonplaats', 50);
 		$fields[] = new RequiredLandField('land', $profiel->land, 'Land');
 
-		if (!$profiel->isOudlid()) {
+		if ($profiel->propertyMogelijk('o_adres') || $inschrijven) {
```

Comment:
Love it.

Reply:
Impliceert `$inschrijven` niet `propertyMogelijk`?

Reply:
Nee, propertyMogelijk kijkt alleen naar de lidstatus. Bij inschrijven is er nog geen lidstatus